aea/registries/base.py

```diff
@@ -468,19 +469,15 @@ def __init__(self):
 
         self._registries = [self.protocol_registry, self.handler_registry, self.behaviour_registry, self.task_registry]
 
-    @classmethod
-    def from_resource_dir(cls, directory: str, agent_context: AgentContext) -> Optional['Resources']:
```

Comment:
this was an obsolete method. The suggested approach now would be:
```python
resources = Resources(directory_path)
aea = AEA(..., resources=resources)
resources.load(aea.context)
```

Reply:
notice, however, that `resources.load` is not mandatory.
It is called by the framework on `aea.start()`.

Reply:
Yes, so we should probably not add the `resources.load(area.context)` line to the demos.

Reply:
yes, agreed.